### ISSUE TITLE
Root directory marked unavailable

### ISSUE THREAD

**Issue by JohnMH** [lidarr/Lidarr#1728]
**Describe the bug**
Running Lidarr in a FreeBSD 11.4 jail with Mono 5.20.1.34, I cannot add a root folder and have it marked as anything other than 'unavailable'. The mountpoint I have for Lidarr is mounted as type `nullfs`.

**To Reproduce**
Add a root folder.
It's marked as unavailable.
**Expected behavior**
The root folder is added, and is available.

**Logs**
20-11-6 01:24:33.4|Trace|Http|Req: 40 [POST] /api/v1/rootFolder
20-11-6 01:24:34.1|Trace|EventAggregator|Publishing ModelEvent<RootFolder>
20-11-6 01:24:34.1|Trace|EventAggregator|ModelEvent<RootFolder> -> RootFolderModule
20-11-6 01:24:34.1|Debug|DiskProvider|path: /mnt/REDACTED
20-11-6 01:24:34.1|Debug|DiskProvider|Unable to get free space for '/mnt/REDACTED', unable to find suitable drive
20-11-6 01:24:34.1|Debug|RootFolderService|Generating list of unmapped folders
20-11-6 01:24:34.1|Debug|RootFolderService|37 unmapped folders detected.
20-11-6 01:24:34.1|Trace|EventAggregator|ModelEvent<RootFolder> <- RootFolderModule
20-11-6 01:24:34.1|Debug|DiskProvider|path: /mnt/REDACTED
20-11-6 01:24:34.1|Debug|DiskProvider|Unable to get free space for '/mnt/REDACTED', unable to find suitable drive
20-11-6 01:24:34.1|Debug|RootFolderService|Generating list of unmapped folders
20-11-6 01:24:34.1|Trace|Http|Req: 41 [GET] /api/v1/rootFolder
20-11-6 01:24:34.1|Debug|DiskProvider|path: /mnt/REDACTED
20-11-6 01:24:34.1|Debug|DiskProvider|Unable to get free space for '/mnt/REDACTED', unable to find suitable drive
20-11-6 01:24:34.1|Debug|RootFolderService|37 unmapped folders detected.
20-11-6 01:24:34.1|Debug|RootFolderService|Generating list of unmapped folders
20-11-6 01:24:34.1|Debug|DiskProvider|path: /mnt/REDACTED
20-11-6 01:24:34.1|Debug|DiskProvider|Unable to get free space for '/mnt/REDACTED', unable to find suitable drive
20-11-6 01:24:34.1|Debug|RootFolderService|Generating list of unmapped folders
20-11-6 01:24:34.1|Debug|RootFolderService|37 unmapped folders detected.
20-11-6 01:24:34.1|Trace|Http|Res: 41 [GET] /api/v1/rootFolder: 200.OK (11 ms)
20-11-6 01:24:34.1|Debug|Api|[GET] /api/v1/rootFolder: 200.OK (11 ms)
20-11-6 01:24:34.1|Debug|RootFolderService|37 unmapped folders detected.
20-11-6 01:24:34.2|Trace|Http|Res: 40 [POST] /api/v1/rootFolder: 201.Created (762 ms)
20-11-6 01:24:34.2|Debug|Api|[POST] /api/v1/rootFolder: 201.Created (762 ms)

**System info (please complete the following information):**
 - Lidarr Version: 0.7.2.1878
 - Operating System: FreeBSD 11.4 (jail), running Lidarr Linux tarball
 - .net Framework (Windows) or mono (macOS/Linux) Version: Mono 5.20.1.34

**Comment by JohnMH** [comment]
For reference, it's clearly not a permissions issue, because it shows the number of unmapped folders inside.

**Comment by GunbleR** [comment]
same. 
i still can remove the dir, add it again and get import screen once. after that its written as unavailable again.
"0 unmapped folders detected" - currently all folders inside are mapped, it does show unmapped folders when the exist.

I'm using freenas, currently 11.3-U5.
jail is 11.4
directory mounted as nullfs from pool that also smb-shared
lidarr Version
    0.7.2.1878
Mono Version
    5.20.1.34


```
20-11-14 11:27:38.9|Trace|Http|Req: 1238 [GET] /api/v1/rootFolder
20-11-14 11:27:38.9|Debug|DiskProvider|path: /media/music/
20-11-14 11:27:38.9|Debug|DiskProvider|Unable to get free space for '/media/music/', unable to find suitable drive
20-11-14 11:27:38.9|Debug|RootFolderService|Generating list of unmapped folders
20-11-14 11:27:38.9|Debug|RootFolderService|0 unmapped folders detected.
20-11-14 11:27:38.9|Trace|Http|Res: 1238 [GET] /api/v1/rootFolder: 200.OK (38 ms)
20-11-14 11:27:38.9|Debug|Api|[GET] /api/v1/rootFolder: 200.OK (38 ms)
20-11-14 11:27:43.8|Trace|Scheduler|Pending Tasks: 0

```

**Comment by GunbleR** [comment]
same issue on sonarr v3 with fix
https://github.com/Sonarr/Sonarr/issues/3275

**Comment by stynoo** [comment]
Same issue, same setup.   It looks like this issue was fixed and merged before: [commit here](https://github.com/lidarr/Lidarr/commit/3743b023f2e3105309d8101df03d61813edf398e)

**Comment by svenEsven** [comment]
I've been banging my head against this wall forever now.

**Comment by ta264** [comment]
This is fixed in 0.8
